packages/cli/lib/pipeline.js

```diff
@@ -38,7 +48,17 @@ export const desugar = async (dataset, { logger, knownOperations } = {}) => {
         continue
       }
 
-      const { type, link } = knownStep
+      let { type, link } = knownStep
+      const { pathname, filename, hash, protocol } = iriResolve(link.value, pipelinePath)
+
+      if (protocol === 'node:' && isInstalledGlobally) {
+        // must resolve node: links to absolute paths to overcome node resolution issues in global context
+        const resolved = require.resolve(pathname)
+        if (filename !== resolved) {
+          link = rdf.namedNode(url.pathToFileURL(resolved).toString() + hash)
+        }
+      }
```

Comment:
And then, in global context, `node:` implementations of steps must be replaced with absolute `file:` URLs.

Somehow this cannot be done uniformly in local installation...